developer-extension/src/panel/useSdkConfig.ts

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from 'react'
+
+export function useSdkConfig() {
+  const [rumConfig, setRumConfig] = useState<object>()
+  const [logsConfig, setLogsConfig] = useState<object>()
+
+  useEffect(() => {
+    retry(getInitConfiguration('rum')).then(setRumConfig).catch(console.error)
+    retry(getInitConfiguration('logs')).then(setLogsConfig).catch(console.error)
+  }, [])
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
Maybe keep a 2s refresh like in the previous version, so it'll refresh when we navigate to another page, and won't need to `retry` anymore

```suggestion
  useEffect(() => {
    function getInitConfigurations() {
      getInitConfiguration('rum').then(setRumConfig).catch(console.error)
      getInitConfiguration('logs').then(setLogsConfig).catch(console.error)
    }
    const id = setInterval(getInitConfigurations, 2000)
    getInitConfigurations()
    return () => clearInterval(id)
  }, [])
```

Reply:
yes